rcon/blacklist.py

```diff
@@ -558,6 +600,23 @@ def edit_record_from_blacklist(
                 )
             )
 
+        # temp ban -> perma ban
+        if old_record["expires_at"] and not new_record["expires_at"]:
+            update_penalty_count(
+                player_id=record.player.player_id,
+                player=record.player,
+                action=PlayerActionState.TEMPBAN,
```

Comment:
Shouldn't this be `PERMABAN`? And the instance further below `TEMPBAN`?

Reply:
Whoops! Fixed